docs/dev/docker-compose.yaml

```diff
@@ -28,8 +28,9 @@ services:
       - "4223:4223"
       - "8223:8223"
   pulsar:
-    image: apachepulsar/pulsar:2.10.0
-    profiles: ["linux"]
+    image: kezhenxu94/pulsar
```

Comment:
Why the change? also, can we remove the commented code?

Reply:
UGH! This absolutely was not meant to be included in this diff (heat of the moment hacking here). I"ll remove it.

Reply:
This change excised in commit 91f5ccb6